lms/djangoapps/grades/new/course_grade.py

```diff
@@ -336,7 +336,7 @@ def create(self, student, course, read_only=True):
             raise PermissionDenied("User does not have access to this course")
         return (
             self._get_saved_grade(student, course, course_structure) or
-            self._compute_and_update_grade(student, course, course_structure, read_only)
+            self._compute_and_update_grade(student, course, course_structure, read_only=read_only)
```

Comment:
I think this is just a relic from a previous attempt at implementing the event transaction ID as  a pass-through arg. I'll revert it.